cloudvolume/cloudvolume.py

```diff
@@ -178,16 +181,75 @@ def refresh_info(self):
         self.info = info
         return self.info
 
+    self.info = self._fetch_info()
+    self._maybe_cache_info()
+    return self.info
+
+  def _check_cached_info_validity(self):
+    """
+    ValueError if cache differs at all from source data layer with
+    an excepton for volume_size which prints a warning.
+    """
+    cache_info = self._read_cached_json('info')
+    if not cache_info:
+      return
+
+    fresh_info = self._fetch_info()
+
+    mismatch_error = ValueError("""
+      Data layer info file differs from cache. Please check whether this
+      change invalidates your cache. 
+
+      If VALID do one of:
+        1) Manually delete the cached info file (see location below)
+        2) Refresh your on-disk cache as follows:
+          vol = CloudVolume(..., cache=False) # refreshes from source
+          vol.cache = True
+          vol.commit_info() # writes to disk
+      If INVALID do one of: 
+        1) Delete the info file manually (see cache location below) 
```

Comment:
This should probably say 'Delete the cache folder'. If the cache info file is invalid, most likely the cache files themselves are invalid as well.